Overview.bs

```diff
@@ -2081,15 +2081,52 @@ id.
 
 <h2 class=no-num id=priv>Privacy Considerations</h2>
 
-Coming soon.
 
-<!-- TODO: content from the previous spec should be a good starting point, but will need changes. -->
+<h3  id="content-inference-from-character-set">Content inference from character set</h3>
 
-<h2 class=no-num id=sec>Security Considerations</h2>
+IFT exposes, to the server hosting a Web font, information on the set of characters that the browser wants to render with the font (for
+details, see [[#extending-font-subset]]).
 
-Coming soon.
+For some languages, which use a very large character set (Chinese and Japanese are examples) the vast reduction in total
+bytes transferred means that Web fonts become usable, including on mobile networks, for the first time. However, for those languages, it
+is <em>possible</em> that individual requests might be analyzed by a rogue font server to obtain intelligence about the type of content
+which is being read. It is unclear how feasible this attack is, or the computational complexity required to exploit it, unless the
+characters being requested are very unusual.
+
+More specifically, a IFT font includes a collection of unicode codepoint groups and requests will be made for groups that intersect
+content being rendered. This provides information to the hosting server that at least one codepoint in a group was needed, but does not
+contain information on which specific codepoints within a group were needed. This is functionally quite similar to the existing
+[[css-fonts-4#unicode-range-desc]] and has the same privacy implications. Discussion of the privacy implication of unicode-range can be
+found in the CSS Fonts 4 specification:
+
+*  [[css-fonts-4#sp201]]. For especially privacy-sensitive-contexts this recommends the user agent download all web fonts in a document.
+    For IFT fonts, utilizing [[#fully-expanding-a-font]] will fetch the entire available IFT font without providing any information about
+    the specific content present. Alternatively, in a privacy sensitive contexts a user agent could randomly select additional patches
+    that are not required by the current content to provide obfuscation of what patches are actually needed.
+
+*  [[css-fonts-4#sp208]]
+
+<h3 id="per-origin">Per-origin restriction avoids fingerprinting</h3>
+
+  As required by [[!css-fonts-4]],
+  Web Fonts [[css-fonts-4#web-fonts|must not be accessible
+  in any other Document from the one which either is associated with the @font-face rule
+  or owns the FontFaceSet]].
+  Other applications on the device must not be able to access Web Fonts.</a>
```

Comment:
This "Other applications" sentence is (arguably) redundant with the negative phrasing but not with the positive phrasing I'm endorsing. So I think it should go if the above stays as it is but stay if it switches.

Reply:
This is also a direct quote from the css 4 doc. I've reworked the formatting in this section to make it clear what's been quoted directly.